addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -123,8 +123,8 @@ class Applicant(models.Model):
     salary_expected = fields.Float("Expected Salary", group_operator="avg", help="Salary Expected by Applicant", tracking=True)
     availability = fields.Date("Availability", help="The date at which the applicant will be available to start working", tracking=True)
     partner_name = fields.Char("Applicant's Name")
-    partner_phone = fields.Char("Phone", size=32, compute='_compute_partner_phone_email', store=True, readonly=False)
-    partner_mobile = fields.Char("Mobile", size=32, compute='_compute_partner_phone_email', store=True, readonly=False)
+    partner_phone = fields.Char("Phone", size=32, compute='_compute_partner_phone_email', inverse='_inverse_partner_phone', store=True, readonly=False)
+    partner_mobile = fields.Char("Mobile", size=32, compute='_compute_partner_phone_email', inverse='_inverse_partner_mobile', store=True, readonly=False)
```

Comment:
Remove the `readonly=False`, computed fields with inverse are `readonly=False` ;).